### PR TITLE
integration test v2: report snapshot status after sending

### PR DESCRIPTION
**Pull request by SpadeA-Tang**
### What is changed and how it works?

Issue Number: Ref #12842

What's Changed:

```commit-message
report snapshot status after sending
```

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/test_raftstore-v2/src/node.rs [hunk 1]
```diff
@@ -102,7 +104,14 @@ impl Transport for ChannelTransport {
         match core.routers.get(&to_store) {
             Some(h) => {
                 h.send_raft_msg(msg)?;
-                // report snapshot status if needed
+                if is_snapshot {
+                    // should report snapshot finish
```
Comment (tabokie) [inline components/test_raftstore-v2/src/node.rs:108]: Useless comment.

**Review by tabokie — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Comment by bufferflies** [comment]
needs to edit the pr's description.

**Review by AndreMouche — APPROVED** [review]
LGTM

**Pull request merged**